src/components/NavBar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { FaUserCircle } from 'react-icons/fa';
+import { Link } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const NavBar = () => (
+  <nav className="container d-flex align-items-center bg-light py-4 shadow justify-content-evenly">
+    <div className="me-auto px-5 d-flex align-items-center">
+      <Link className="fw-bold text-decoration-none fs-3" style={{ color: '#0290FF' }} to="/">Bookstore CMS</Link>
```

Comment:
- [ ] Let's try to avoid inline styling as it is not best practice. Kindly move all styles to an external CSS file.